cachito/workers/utils.py

```diff
@@ -1,17 +1 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
Any reasons for not removing the entire module?

Reply:
Just thought `utils.py` might be also used to include other utility code in the future.

Reply:
It's easy enough to add it back in. I'd prefer if we removed it.

Reply:
Done. The empty files are removed.